src/main/java/com/cloudbees/jenkins/plugins/BitbucketJobProbe.java

```diff
@@ -65,6 +68,15 @@ public void triggerMatchingJobs(String user, String url, String scm) {
         }
     }
 
+    private boolean hasBeenTriggered(List<SCM> scmTriggered, SCM scmTrigger) {
+        for (SCM scm : scmTriggered) {
+            if (scm.equals(scmTrigger)) {
```

Comment:
It seems to work, but not sure why. At any rate it must be fixed to use something similar to `match` (using `RemoteConfig remoteConfig : ((GitSCM) scm).getRepositories()`).

BTW this plugin is not compatible with Mercurial. But I think it would be a new feature (out of scope here).